src/ui/public/query_bar/directive/query_bar.html

```diff
@@ -14,7 +14,7 @@
           disable-input-focus="queryBar.disableAutoFocus"
           kbn-typeahead-input
           ng-model="queryBar.localQuery.query"
-          placeholder="Search... (e.g. status:200 AND extension:PHP)"
+          ng-attr-placeholder="{{queryBar.searchPlaceholder}}"
```

Comment:
Since we know, that this can't change while the template is loaded, please use bind-once syntax: `{{::queryBar.searchPlaceholder}}`

Reply:
Done